### PR TITLE
[Translation] fix: add nikic/php-parser requirement for translation:extract in tran…

### PR DESCRIPTION
**Pull request by bsramin**
…slation.rst

### CODE REVIEW

**Comment by bsramin** [comment]
Find this from this blog news

https://symfony.com/blog/new-in-symfony-6-2-better-php-translation-extractor

**Comment by MrYamous** [comment]
Can be applied on 6.4 too I think

File: translation.rst [hunk 1]
```diff
@@ -457,6 +457,16 @@ The ``translation:extract`` command looks for missing translations in:
 * Any PHP file/class stored in the ``src/`` directory that uses
   :ref:`Constraints Attributes <validation-constraints>`  with ``*message`` named argument(s).
 
+To read PHP files, is used the new ``PhpAstExtractor`` service supports all kinds of trans() function calls, usages of TranslatableMessage class, messages defined in validation constraints, etc...
```
Comment (MrYamous) [inline translation.rst]: We may can add a `.. versionadded:: 6.2 ` directive if applied on 6.4 branch

Comment (MrYamous) [inline translation.rst]:
```suggestion
To read PHP files, is used the new ``PhpAstExtractor`` service supports all kinds of ``trans()`` function calls, usages of :class:`Symfony\\Component\\Translation\\TranslatableMessage` class, messages defined in validation constraints, etc...
```

File: translation.rst [hunk 2]
```diff
@@ -457,6 +457,16 @@ The ``translation:extract`` command looks for missing translations in:
...
+
+To use the new translation extractor, install the ``nikic/php-parser`` package using Composer, before using `translation:extract`.
+
```
Comment (MrYamous) [inline translation.rst:466]:
```suggestion
```

File: translation.rst [hunk 3]
```diff
@@ -457,6 +457,16 @@ The ``translation:extract`` command looks for missing translations in:
...
+
+.. code-block:: terminal
+
+    $ composer require nikic/php-parser
+
+
```
Comment (MrYamous) [inline translation.rst]:
```suggestion
```

**Comment by javiereguiluz** [comment]
@bsramin this is now merged! Thanks ... and congrats on your first Symfony Docs contribution 🎉 

Note that while merging we did some reword (see https://github.com/symfony/symfony-docs/commit/ad9262617b9bd02027f840c3626470725112e02c). Don't worry, this is normal for Symfony Docs contributions. We do that to ensure that all docs have the same tone and style.

In this case, we moved the new docs to a `.. tip` directive to make it stand out. Also, we removed some of the most technical details because that might be OK for a blog post but it's a little too much detail for the docs.

In any case, thanks again for your contribution!

**Comment by bsramin** [comment]
Thanks 👍🏻

**Pull request merged**